src/main/java/core/basesyntax/FileReader.java

```diff
@@ -0,0 +1,261 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Feel free to remove this class and create your own.
+ * <p>
+ * jv-file-editor
+ * Написать небольшое консольное приложение для работы с текстовыми файлами,
+ * которое будет обладать следующими командами:
+ * <p>
+ * create [path] [file-name]
+ * Создает текстовый файл по указанному пути. Если путь не существует,
+ * вывести соответствующее сообщение. Если файл уже существует, вывести
+ * запрос на его перезапись
+ * <p>
+ * read [path] [file-name]
+ * Считывает файл по указанному пути и выводит текст в консоль. Если
+ * указанного пути и/или файла не существует, вывести соответствующее сообщение
+ * <p>
+ * info [path] [file-name]
+ * Выводит краткую информацию по указанному файлу: количество символов, строк,
+ * слов, дату и время последнего изменения, размер файла
+ * <p>
+ * help
+ * Выводит в консоль все доступные комманды и информацию к ним
+ * <p>
+ * help [command]
+ * Выводит в консоль информацию по указанной команде
+ * <p>
+ * exit
+ * Завершение работы программы
+ * <p>
+ * Если написать в консоль любой текст, который не является командой,
+ * и нажать enter, должно появиться сообщение с предложением сохранить текст
+ */
+
+class FileReader {
+    private static final String PATH_EXAMPLE = "D:\\\\java\\\\file-editor";
+    private static final String FILE_EXAMPLE = "file-editor";
+    private static List<String> commandList = new ArrayList<>(
+            Arrays.asList("create", "read", "info", "help", "help c", "exit"));
+    private String filePath = "";
+    private String fileName = "";
+
+    private String input() {
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        try {
+            return reader.readLine();
+        } catch (IOException e) {
+            throw new RuntimeException("IOException", e);
```

Comment:
add a more meaningful message
```suggestion
            throw new RuntimeException("Can't read command", e);
```
or smth like this